app/views/AttachmentView.tsx

```diff
@@ -87,17 +89,22 @@ class AttachmentView extends React.Component<IAttachmentViewProps, IAttachmentVi
 			// Do nothing
 		}
 		const options = {
-			title,
-			headerLeft: () => <HeaderButton.CloseModal testID='close-attachment-view' navigation={navigation} />,
-			headerRight: () =>
-				Allow_Save_Media_to_Gallery ? <HeaderButton.Download testID='save-image' onPress={this.handleSave} /> : null,
-			headerBackground: () => <View style={{ flex: 1, backgroundColor: themes[theme].previewBackground }} />,
-			headerTintColor: themes[theme].previewTintColor,
-			headerTitleStyle: { color: themes[theme].previewTintColor, paddingHorizontal: 20 },
-			headerTitleContainerStyle: { marginHorizontal: -20 },
-			headerTitleAlign: 'center'
+			header: () => (
+				<Header
+					title={title || ''}
+					headerTintColor={themes[theme].previewTintColor}
+					headerStatusBarHeight={isTablet ? 20 : 5}
```

Comment:
I only had to add this, because the modal with the `header` from elements has this behavior: 
![Screen Shot 2022-06-24 at 00 05 59](https://user-images.githubusercontent.com/14894380/175454085-ef9db39b-dc72-4e1d-b4e2-2b350dcc8dc9.png)

So I added the `headerStatusBarHeight` to get the title closer to the top. I can remove it this if I add this props `{...TransitionPresets.ModalSlideFromBottomIOS }` to the options. We'll still have the modal effect, but the modal will use all screen size, like we have today. 

![Screen Shot 2022-06-24 at 00 08 49](https://user-images.githubusercontent.com/14894380/175454404-428d016e-bdb1-4511-a054-80596f0c29f8.png)

Or we may not use the `Header` from elements, but we'll have the problem that I explained bellow.